Web/src/components/common/VirtualizedList.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {
+  List,
+  Index,
+  WindowScroller,
+  ListRowRenderer,
+  AutoSizer,
+} from 'react-virtualized';
+
+type VirtualizedListProps = {
+  overscanRowCount?: number;
+  noRowsRenderer?: () => JSX.Element;
+  rowCount: number;
+  rowHeight:
+  | number
+  | (number & ((info: Index) => number))
+  | (((params: Index) => number) & number)
+  | (((params: Index) => number) & ((info: Index) => number));
+  rowRenderer: ListRowRenderer;
+  scrollToIndex?: number;
+};
+
+const WindowScrollVirtualizedList: React.FC<VirtualizedListProps> = ({
```

Comment:
Perhaps what you want is `import { ListProps } from 'react-virtualized';`
And then
```suggestion
const WindowScrollVirtualizedList: React.FC<ListProps> = ({
```

Or if you want to limit the keys, can use this https://www.typescriptlang.org/docs/handbook/utility-types.html#picktk